drivers/node/ssh/ssh.go

```diff
@@ -194,6 +194,17 @@ func (s *SSH) Init(nodeOpts node.InitOptions) error {
 	return nil
 }
 
+// Init initializes SSH node driver
+func (s *SSH) Init(nodeOpts node.InitOptions) error {
+	s.specDir = nodeOpts.SpecDir
+	return s.updateDriver()
+}
+
+// Refreshes the driver on a context switch
+func RefreshDriver(s *SSH) error {
```

Comment:
I think the function name "RefreshDriver" may not be appropriate as it suggests updating the drivers, whereas in this case we are actually inspecting the drivers.